### PR TITLE
Datatype of game name changed

### PR DESCRIPTION
**Pull request by CCR-bhurtel**
Datatype of game was changed from INTEGER to STRING

### CODE REVIEW

File: versioned_docs/version-6.x.x/advanced-association-concepts/advanced-many-to-many.md [hunk 1]
```diff
@@ -545,7 +545,7 @@ const sequelize = new Sequelize('sqlite::memory:', {
 });
 const Player = sequelize.define('Player', { username: DataTypes.STRING });
 const Team = sequelize.define('Team', { name: DataTypes.STRING });
-const Game = sequelize.define('Game', { name: DataTypes.INTEGER });
+const Game = sequelize.define('Game', { name: DataTypes.STRING });
```
Comment (WikiRik) [inline versioned_docs/version-6.x.x/advanced-association-concepts/advanced-many-to-many.md:548]: We need to update it in line 487 as well

**Comment by WikiRik** [comment]
Can you update it for the v7 docs as well? https://github.com/sequelize/website/blob/main/docs/advanced-association-concepts/advanced-many-to-many.md

**Comment by CCR-bhurtel** [comment]
done

**Review by ephys — APPROVED** [review]
Thanks!

**Pull request merged**